src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
```

Comment:
```suggestion
  constructor(name, health = 100) {
```
You can set value by default like this